.github/scripts/deploy.sh

```diff
@@ -13,6 +18,7 @@ aws s3api put-object --bucket $bucket --key "content/$version/LICENSE" --body LI
 if [[ $version =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]
 then
     echo $version | aws s3 cp - s3://$bucket/content/current
+    echo "$allVersions" | aws s3 cp - s3://$bucket/content/versions
```

Comment:
issue: To prevent race conditions, I think we should update the `current` and `versions` files after deploying the new version (i.e., immediately before the `fi` statement at line 34).

Reply:
Refactored